src/EncompassRest/LoanPipeline/LoanPipelineCursor.cs

```diff
@@ -23,59 +23,53 @@ internal LoanPipelineCursor(EncompassRestClient client, string cursorId, int cou
             Fields = fields ?? Array<string>.Empty;
         }
 
-        public Task<LoanPipelineData> GetItemAsync(int index) => GetItemAsync(index, CancellationToken.None);
+        public Task<LoanPipelineData> GetItemAsync(int index) => GetItemAsync(index, null, CancellationToken.None);
 
-        public async Task<LoanPipelineData> GetItemAsync(int index, CancellationToken cancellationToken)
+        public Task<LoanPipelineData> GetItemAsync(int index, IEnumerable<string> itemFields) => GetItemAsync(index, itemFields, CancellationToken.None);
+
+        public async Task<LoanPipelineData> GetItemAsync(int index,IEnumerable<string> itemFields, CancellationToken cancellationToken)
```

Comment:
add space between parameters

I think `itemFields` ought to be named just `fields`. Thoughts?